packages/mobile-app/app/(drawer)/account/send/index.tsx

```diff
@@ -325,7 +325,8 @@ export default function Send() {
                   appearance="filled"
                   style={styles.confirmButton}
                   onPress={() => {
-                    console.log("will navigate to", sentTxHash);
+                    setTransactionState("idle");
+                    router.replace(`/transaction/${sentTxHash}`);
```

Comment:
Hmm, should we nest the tx in the account stack? Don't feel strongly about it, it just feels like it fits better there